### PR TITLE
Adds the add-corporation action

### PR DESCRIPTION
**Pull request by lazypower**

### CODE REVIEW

File: reactive/vpe_router.py [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-)
-
-from charms.reactive import (
-    hook,
-    set_state,
-    is_state,
-    remove_state,
-    main,
-)
+from charmhelpers.core.hookenv import config
```
Comment (marcoceppi) [inline reactive/vpe_router.py:2]: why remove the import parens here?

**Comment by lazypower** [comment]
It angers Flake8 as an invalid import format

**Comment by marcoceppi** [comment]
that's new. @tvansteenburgh has pep8 changed on the "right way" to do multiple imports from a single module?

**Comment by lazypower** [comment]
Well this is flake8, not pep8. Which probably makes a difference

**Comment by marcoceppi** [comment]
pep8 is the style standard for Python, flake8 combines pyflakes, pep8, and some other linting tools into one

**Comment by marcoceppi** [comment]
https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses which may not be the most recent one, but still holds

**Comment by lazypower** [comment]
this was stale, and had a lot of conjecture about the import statements i rewrote so i'm closing this

**Pull request closed without merging**